Sources/MpcNET/Commands/Database/FindAndSearchCommands.cs

```diff
@@ -170,14 +480,27 @@ public BaseFilterCommand(List<KeyValuePair<ITag, string>> filters, int windowSta
         /// Serializes the command.
         /// </summary>
         /// <returns>
-        /// The serialize command.
+        /// The serialized command.
         /// </returns>
         public string Serialize()
         {
-            var serializedFilters = string.Join(" AND ",
-                  filters.Select(x => $"({x.Key.Value} {Operand} {escape(x.Value)})")
-                  );
-            var cmd = $@"{CommandName} ""({serializedFilters})""";
+            string cmd = "";
+
+            if (tagFilters != null)
+            {
+                var serializedFilters = string.Join(" AND ",
+                    tagFilters.Select(x => $"({x.Key.Value} {Operand} {escape(x.Value)})")
+                );
+                cmd = $@"{CommandName} ""({serializedFilters})""";
+            }
+
+            if (completeFilters != null)
```

Comment:
Adding some tests to the existing Test project using `FindCommand` would also be nice.  
(I'm aware the search commands don't have any tests currently, so no biggie if you don't feel like writing some)